cosim/black-parrot-example/vcs/Makefile

```diff
@@ -30,13 +30,17 @@ CFLAGS += -DSCRATCHPAD_ENABLE
 #CFLAGS += -DSIM_BACKPRESSURE_LENGTH=100
 CFLAGS += $(INCLUDES) $(SKIP_DRAM)
 
+VCS_OPTS += +define+SIM_MP
```

Comment:
What's this define for?

Reply:
yeah as commented above, separate simulation/synthesis parts of modules into separate files and then compile the files with the correct target (simulation in verilator/vcs, synthesis with fpga)